client/lib/reader-tags/tags.js

```diff
@@ -18,10 +20,12 @@ function receiveTags( newTags ) {
 	forOwn( newTags, ( sub ) => {
 		sub.URL = '/tag/' + sub.slug + '/';
 		sub.title = decodeEntities( sub.title );
-		sub.slug = sub.slug.toLowerCase();
+		sub.slug = (sub.slug) ? sub.slug.toLowerCase() : '';
 		tags[ sub.slug ] = sub;
 	} );
-	TagStore.emit( 'change' );
+	if (!(isArray(newTags) && newTags[0].hasOwnProperty('error_data') && newTags[0].error_data.hasOwnProperty('invalid_tag'))) {
```

Comment:
Can you explain the rationale here - why don't we want to emit the event when there's an invalid tag?